lib/builder/builder.js

```diff
@@ -246,6 +248,10 @@ module.exports = {
 					tasks: projectTasks,
 					project,
 					parentLogger: log
+				}).then(() => {
```

Comment:
For readability, and also to catch any errors that rimraf might raise, rather wrap rimraf using [promisify](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original):
```js
const {promisify} = require("util");
const rimraf = promisify(require("rimraf"));
```

Then use an async function like this:
```js
				}).then(async () => {
					if (clearDestPath) {
						await rimraf(destPath);
                    }
				}).then(() => {
				...
```

Reply:
Wrapped.